cea/technologies/storage_tank.py

```diff
@@ -215,6 +217,15 @@ def calc_tank_temperature(T_start_C, q_loss_W, q_discharged_W, q_charged_W, V_ta
 # cold water storage tank design
 # ================================
 
+def calc_storage_tank_mass(Qc_tank_capacity_Wh,
```

Comment:
should it be names `calc_storage_tank_volume` instead?

Reply:
This function calculates the weight of the phase change material in the tank. Since the material's phase changes as the storage is charged and discharged, so does its density and therefore its volume. So I would say the current name is more accurate.

Reply:
i think we are referring to two different functions with the same name.
you are describing `calc_storage_tank_mass` in `storage_tank_pcm.py`, and i am referring to `storage_tank.py`
anyways, i can resolve that part